uxarray/helpers.py

```diff
@@ -41,6 +41,8 @@ def parse_grid_type(dataset):
     # ugrid topology
     elif _is_ugrid(dataset):
         mesh_type = "ugrid"
+    elif "verticesOnCell" in dataset:
+        mesh_type = "mpas"
```

Comment:
It looks like the docstring needs to be updated to include MPAS (and maybe remove shape files, which don't in fact seems to be supported yet).

Also, is there any concern that the approach of looking for a specific variable name to detect a given format won't be robust?  `verticesOnCell` seems pretty safe for detecting MPAS but `coord` and `coordx` seem like they could be pretty generic names.  I know from ParaView that it can be frustrating to have files misidentified because the reader's identification of a format is too generic (every NetCDF file gets identified as a CAM atmosphere file for some reason).

Reply:
> It looks like the docstring needs to be updated to include MPAS (and maybe remove shape files, which don't in fact seems to be supported yet).
> 
> Also, is there any concern that the approach of looking for a specific variable name to detect a given format won't be robust? `verticesOnCell` seems pretty safe for detecting MPAS but `coord` and `coordx` seem like they could be pretty generic names. I know from ParaView that it can be frustrating to have files misidentified because the reader's identification of a format is too generic (every NetCDF file gets identified as a CAM atmosphere file for some reason).

Good catch on the docstring. 

I agree that the way that we check what format the grid is in is not the most robust. I'll raise an issue about it today, since having more complete functions to check what format a dataset is in would be better (i.e. `_is_mpas()`, `_is_scrip()`, etc)

Reply:
#238